livre/core/pipeline/Workers.cpp

```diff
@@ -30,9 +30,9 @@ namespace livre
 struct Workers::Impl
 {
     Impl(const std::string& name, Workers& workers, const size_t nThreads,
-         const ConstGLContextPtr& glContext)
+         const GLContext& glContext)
         : _workers(workers)
-        , _glContext(glContext)
+        , _glContext(glContext.clone())
```

Comment:
I don't know it is costly, but it wasn't cloned in the past

Reply:
This whole code needs rewriting.

It needs to be cloned, since if it's done later in the worker thread, the cloned context cannot be current (otherwise Windows says no-no). This code is not ideal, but what can you do...